src/pages/DatasetRegistration.js

```diff
@@ -70,8 +70,6 @@ class DatasetRegistration extends Component {
         havePi: '',
         profileName: '',
         piName: '',
-        urlDAA: '',
```

Comment:
DatasetRegistration has state variable called ```file``` that is also tied to the DAA modes (similar to DataAccessRequestRenewal). Should that also be removed?

Reply:
I think it's safe to remove! A lot of the state in DatasetRegistration won't end up being used. I was going to remove it in my own PR if Greg doesn't do it here.

Reply:
Yes, that should go too, I missed that one

Reply:
Done!